corehq/apps/export/utils.py

```diff
@@ -302,7 +302,10 @@ def convert_saved_export_to_export_instance(
     if isinstance(instance, CaseExportInstance):
         migration_meta.has_case_history = instance.has_case_history_table
 
-    if not dryrun:
+    has_issues = any([migration_meta.skipped_tables,
+                      migration_meta.skipped_columns,
+                      migration_meta.has_case_history])
```

Comment:
I pulled this from here:
https://github.com/dimagi/commcare-hq/blob/a34d196c7742e3d5306b0ba41851993e344723cf/corehq/apps/export/utils.py#L657
You can see from the usage there that if `not has_issues`, there's no need to display output, indicating that the conversion was a success.